R/ruby/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ruby"
+version = v"2.6.6"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource(
+        "https://cache.ruby-lang.org/pub/ruby/2.6/ruby-2.6.6.tar.gz",
+        "364b143def360bac1b74eb56ed60b1a0dca6439b00157ae11ff77d5cd2e92291",
+    ),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/ruby-2.6.6/
+apk add ruby-full
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --with-baseruby=/usr/bin/ruby --enable-shared
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = filter(
+    i -> (i isa Linux && i.libc === :glibc) || i isa FreeBSD,
+    supported_platforms(),
```

Comment:
Stupid but honest question: Why that comma? I understand the usefulness in lists, where it reduces diffs when inserting further list elements in the future; but here it's the argument tuple for `filter`, so we don't expect to insert further arguments below this one with any regularity? What am I missing?
```suggestion
    supported_platforms()
```

Reply:
I just got into the habit of always appending commas at the end of arguments in multiline function calls like this. I think this is overall a little bit more consistent and I believe BlueStyle also requires that.